README.md

```diff
@@ -643,6 +635,40 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+* <a name="no-hanging-indent"></a>
+    Use a new line for the argument list of a method at the end of a chain
+    if they are indented more than one level from the first method.
+<sup>[[link](#no-hanging-indent)]</sup>
+
+  ```ruby
+  # bad (hanging indent)
+  some_method.with.a_longer.chain(:parameter_1,
```

Comment:
I think this needs to be made clear with respect to:

1) no method chain  - e.g. some_method(a,
2) short method chain - frankly it seems to the example you deleted looks fine, as it's not really a long method chain.